packages/shared/src/events/eventing.ts

```diff
@@ -6,7 +6,11 @@ export type EventMetadata = {
 
 export type CommonEventDetails = {
   providerName: string;
+  /**
+   * @deprecated Use `domain` instead
+   */
```

Comment:
```suggestion
  /**
   * @deprecated alias of "domain", use domain instead
   */
```
I think this implies we'll set both so migration is safe.